particles/ShowItems/ShowItems.manifest

```diff
@@ -12,7 +12,8 @@ particle ShowItems in 'ShowItems.js'
   ShowItems(in [~a] list, out [Description] descriptions)
   affordance dom
   affordance dom-touch
-  consume root
+  consume unused  # for testing purposes
```

Comment:
probably shouldn't do this here - maybe create a testing particle with this pattern?

Reply:
done, added test instead.